soc/nxp/mcx/mcxw/soc.c

```diff
@@ -214,4 +215,8 @@ void soc_early_init_hook(void)
 
 	/* restore interrupt state */
 	irq_unlock(oldLevel);
+
+#if defined CONFIG_BT || defined CONFIG_IEEE802154
```

Comment:
use IS_ENABLED macro I think is better with kconfigs here, at the very least use parentheses with defined()